worlds/ladx/ItemIconGuessing.py

```diff
@@ -376,6 +381,22 @@
     # Super Mario World
     'P-BALLOON': 'FEATHER',
 
+    # The Witness
+    'BONK': 'BOMB',
+    'BUNKER LASER': 'INSTRUMENT4',
+    'DESERT LASER': 'INSTRUMENT5',
+    'JUNGLE LASER': 'INSTRUMENT4',
+    'KEEP LASER': 'INSTRUMENT7',
+    'MONASTERY LASER': 'INSTRUMENT1',
+    'PUZZLE SKIP': 'GOLD_LEAF',
+    'QUARRY LASER': 'INSTRUMENT8',
+    'SHADOWS LASER': 'INSTRUMENT1',
+    'SLOWNESS': 'BOMB',
+    'SWAMP LASER': 'INSTRUMENT2',
+    'SYMMETRY LASER': 'INSTRUMENT6',
+    'TOWN LASER': 'INSTRUMENT3',
+    'TREEHOUSE LASER': 'INSTRUMENT2',
```

Comment:
Witness stuff looks good!!!

```suggestion
    'TREEHOUSE LASER': 'INSTRUMENT2',
    'WATER PUMPS': 'KEY',
```

"Swamp Water Pumps" is the only set of doors that does not have "Door" in the name, but it still acts the same as all the other ones, so this would make sense

Reply:
Oh nvm actually there is also 
```suggestion
    'TREEHOUSE LASER': 'INSTRUMENT2',
    'CAVES SHORTCUTS': 'KEY',
    'KEEP SHORTCUTS': 'KEY',
    'MONASTERY SHORTCUTS': 'KEY',
    'SWAMP SHORTCUTS': 'KEY',
```

Unless you think "Shortcuts" is general enough of a word to just add this as a generic